docs/developers/csp.rst

```diff
@@ -84,6 +84,11 @@ do not rely on external CDNs (such as the Formio CDN, Google Fonts...).
 
 * The ``img-src`` requires ``data:``, otherwise map and signature components break.
 
+* If Google Tag manager is enabled, the ``img-src`` requires ``www.googletagmanager.com`` to allow the execution of the
+  Tag Manager container code. More information can be found `here`_.
+
+.. _here: https://developers.google.com/tag-platform/tag-manager/web/csp#enable_the_container_tag_to_use_csp
+
```

Comment:
This should be added to the automatic CSP configuration when activating/deactivating an analytics tool: `src/openforms/analytics_tools/contrib/google_analytics/csp_headers.json`